CMakeLists.txt

```diff
@@ -978,6 +978,15 @@ if(USE_ECH)
   endif()
 endif()
 
+option(USE_SSLS_EXPORT "Enable SSL session export support" OFF)
```

Comment:
It'd be nice to add this option to the list in `docs/INSTALL-CMAKE.md`.

Also to the list of features around line 2050 in `CMakeLists.txt`.
And the same in `configure.ac` to the `SUPPORT_FEATURES` list.

To match the `SSLS-EXPORT` added via `lib/version.c`. This is
checked in CI when this option is enabled. Perhaps it'd be useful
to enable it in an autotools and cmake job.

Reply:
Thanks. Just pushed the additions for CMakeLists.txt and INSTALL-CMAKE.md.

The SUPPORT_FEATURES in configure.ac is already there. Or am I missing something?

Reply:
Thanks! No, sorry, `SUPPORT_FEATURES` is already there indeed, my bad.